Package.swift

```diff
@@ -1,14 +1,14 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7
 import PackageDescription
 
 let package = Package(
 	name: "MQ",
 	platforms: [
-		.iOS(.v13),
-		.macOS(.v11),
-		.macCatalyst(.v13),
-		.tvOS(.v13),
-		.watchOS(.v7),
+		.iOS(.v14),
+		.macOS(.v12),
+		.macCatalyst(.v14),
+		.tvOS(.v14),
+		.watchOS(.v8),
```

Comment:
If possible please change to .watchOS(.v7)

Reply:
updated